### PR TITLE
fix: remove outdated ti.dynamiclib hook, clean up structure

### PR DESCRIPTION
**Pull request by hansemannn**
This pull fixes / refactors a few bits of this module:
- [x] Remove the `ti.dynamiclib` hook which is not required anymore
- [x] Update the install steps that became less painful recently
- [x] Remove the `FRAMEWORK_SEARCH_PATHS` and `LD_RUNPATH_SEARCH_PATHS` flags which are not required anymore

Note: This changes are currently in testing and should be validated on your QE side as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +7,13 @@ PSPDFKit - The Leading Mobile PDF Framework for iOS and Android.
 **Note:** Read the [Alloy](#alloy) section before building the module if you're using the Alloy framework.
 
 1. Checkout the project: `git clone https://github.com/PSPDFKit/Appcelerator-iOS.git`
-2. Make sure you have Titanium 7.5.0.GA or later installed: http://www.appcelerator.com
-3. Download the binary build of PSPDFKit from your customer portal: https://customers.pspdfkit.com
-4. Open the downloaded .dmg and copy `PSPDFKit.framework` and `PSPDFKitUI.framework` into the checked out folder.
-5. Call `appc ti build -p ios --build-only` in the `Appcelerator-iOS` folder.
+2. Make sure you have Titanium 7.5.0.GA or later installed: https://www.appcelerator.com or `ti sdk install 7.5.0.GA`
+3. Download the binary build of PSPDFKit from [your customer portal](https://customers.pspdfkit.com)
+4. Open the downloaded .dmg and copy `PSPDFKit.framework` and `PSPDFKitUI.framework` into the `platform/` folder inside the project.
```
Comment (radazzouz) [inline README.md]: If we change this step, we also need to:

1. Add and track the empty `platform` directory
2. Update the Xcode project.

Currently, if we create this folder and place the frameworks in there, the bindings are not building. Here's the build error:

```sh
[TRACE] [xcode-sim] /Volumes/PSPDFKit/git/Appcelerator-iOS/ComPspdfkit_Prefix.pch:7:9: fatal error: 'PSPDFKit.h' file not found
[TRACE] [xcode-sim] #import "PSPDFKit.h"
[TRACE] [xcode-sim]         ^~~~~~~~~~~~
[TRACE] [xcode-sim] 1 error generated.
[ERROR] [xcode-dist] ** BUILD FAILED **
[ERROR] [xcode-dist] The following build commands failed:
[ERROR] [xcode-dist] 	ProcessPCH build/SharedPrecompiledHeaders/ComPspdfkit_Prefix-bwvovpqnyekuntfgjmjfpishlypg/ComPspdfkit_Prefix.pch.pch ComPspdfkit_Prefix.pch normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
[ERROR] [xcode-dist] 	ProcessPCH build/SharedPrecompiledHeaders/ComPspdfkit_Prefix-hgdtlbofewyanwfznuiwelnggndh/ComPspdfkit_Prefix.pch.pch ComPspdfkit_Prefix.pch normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
[ERROR] [xcode-dist] (2 failures)
```

**Comment by hansemannn** [comment]
The umbrella header needs to be re-referenced as well, I'll do that tomorrow. Also, we could move to CocoaPods or what ever is most easy for you guys.

**Comment by hansemannn** [comment]
@radazzouz Pushed the changes, can you retry? Thanks!

File: README.md (hunk 1 above)
Comment (radazzouz) [reply README.md]: This is no longer an issue. Thanks for addressing it @hansemannn ! 👍

**Comment by radazzouz** [comment]
I was able to successfully build this module now from this branch. However, I am getting an Xcode build error (without much info) when I use the newly created module in an existing sample app.


``` sh
...
[INFO] :   Invoking xcodebuild
[ERROR] :  ** BUILD FAILED **

```

This same sample project works great when using the module built from master.

**Comment by radazzouz** [comment]
I retested this branch and it all works great! Merging! Thank you so much for the contribution, @hansemannn! 👏

**Pull request merged**